utilities/blob_db/blob_db_impl.cc

```diff
@@ -2055,7 +2055,7 @@ Status DestroyBlobDB(const std::string& dbname, const Options& options,
                                         : bdb_options.blob_dir;
 
   std::vector<std::string> filenames;
-  env->GetChildren(blobdir, &filenames);
+  env->GetChildren(blobdir, &filenames).PermitUncheckedError();
```

Comment:
I think you should return the error here.

Reply:
It is legal for this to return an error, for example, if the directory does not exist.

Changed it to only delete the children if this returns ok.